ngraph/core/include/ngraph/function.hpp

```diff
@@ -138,6 +144,24 @@ namespace ngraph
         bool evaluate(const HostTensorVector& output_tensors,
                       const HostTensorVector& input_tensors) const;
 
+        /// Return a list of function's sinks.
```

Comment:
/// Return a list of function's sinks. -> /// \brief Return a list of function's sinks.

Reply:
fixed